### DISCUSSION TITLE
Timetable for the release of CF-1.12

### DISCUSSION THREAD

**Discussion by davidhassell** [cf-convention/discuss#382] [Announcements relevant to CF]
Hello,

At the [CF workshop](http://cfconventions.org/Meetings/2024-Workshop.html) in September, it was agreed that the release of CF-1.12 will be sometime during the period Tuesday 3rd to Friday 6th December 2024.

For a change to make it into this release it must therefore have completed its three week cooling-off by this time i.e. the proposal must be agreed to be complete and accepted by **23:59Z Monday 11th November 2024**, and for there to be no new substantive objections in the following three weeks (changes such as fixing spelling mistakes are still ok, though).

Many thanks,
David

**Comment by JonathanGregory** [comment]
I have added a new temporary label in the conventions repo to all of the open issues which I think we have a good chance of concluding in time for CF1.12, because they already have a PR ready, or preparing one would not be much work. There are [twelve of them](https://github.com/cf-convention/cf-conventions/issues?q=is%3Aissue+is%3Aopen+label%3ACF1.12%3F+).
